pim/utils.py

```diff
@@ -149,10 +150,10 @@ class spinner(threading.Thread):
     """
     def __init__(self):
         if os.name == 'posix':
-            self.chars = (unicodedata.lookup('FIGURE DASH'),u'\\ ',u'| ',u'/ ')
```

Comment:
This all looks reasonable to me, tbh I grabbed the unicode handling bit from a SO post so I'm not 100% confident about how important it was. I was planning to move this `spinner` out into a separate module anyway, so can debug it further at that point if needed.